reflex-dom/src/Reflex/Dom/Internal.hs

```diff
@@ -70,10 +84,13 @@ import System.IO
 import Language.Javascript.JSaddle (JSM)
 
 run :: JSM () -> IO ()
-run jsm = do
+run = runAndroid def
+
+runAndroid :: ActivityCallbacks -> JSM () -> IO ()
+runAndroid activityCallbacks jsm = do
   hSetBuffering stdout LineBuffering
   hSetBuffering stderr LineBuffering
-  continueWithCallbacks $ def
+  continueWithCallbacks $ activityCallbacks
     { _activityCallbacks_onCreate = \_ -> do
```

Comment:
We should probably call the passed-in `onCreate` after we're done with ours.

Reply:
I'm not sure whether that's a good idea, but need to think about the implications of throwing away the passed-in `onCreate` here.